src/ParkDetails/ParkDetails.css

```diff
@@ -0,0 +1,64 @@
+.park-details-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 20px;
+}
+  
+.park-details-title {
+    margin-bottom: 20px;
+    flex: 1;
+    border: 1px solid #090909;
+    background-color: antiquewhite;
+    margin: 10px;
+    text-align: center;
+    border-radius: 10px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.7);
+    padding: 15px;
+}
+  
+.single-title {
+    font-size: 24px;
+}
+  
+.park-info {
+    display: flex;
+    width: 100%;
+}
+  
+.park-details-activities,
+.park-details-weather {
+    flex: 1;
+    border: 1px solid #090909;
+    background-color: antiquewhite;
+    margin: 10px;
+    text-align: center;
+    border-radius: 10px;
```

Comment:
this looks fantastic, ill do a few accessibility scans to make sure it passes all of our testing,  responsive design looks really easy with tailwind, do you want to save responsive design for part 2? should i ask jeremiah that?

Reply:
The app is already fairly responsive. I have grid on auto-fill so it will fit the requirements for it. I wouldn't mind do responsive design for part 2 since its not on our MVP. It'll give us a chance to make it look really nice.